app/main.py

```diff
@@ -16,4 +16,18 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "immutable": [
+        lucky_number,
+        pi,
+        one_is_a_prime_number,
+        name,
+        profile_info
+    ],
+    "mutable": [
+        my_favourite_films,
+        marks,
+        collection_of_coins
```

Comment:
The `collection_of_coins` is a set, which is a mutable data type in Python. It is correctly placed in the 'mutable' list.